src/main/scala/beam/agentsim/agents/ridehail/repositioningmanager/InverseSquareDemandFollowingRepositioning.scala

```diff
@@ -0,0 +1,200 @@
+package beam.agentsim.agents.ridehail.repositioningmanager
+
+import beam.agentsim.agents.ridehail.RideHailManager
+import beam.agentsim.agents.ridehail.RideHailVehicleManager.RideHailAgentLocation
```

Comment:
should we call the file/algo: InverseSquareDistanceDemandFollowingRepositioning?

Reply:
ok